.yamato/com.unity.ml-agents-test.yml

```diff
@@ -24,6 +31,22 @@ test_platforms:
     flavor: b1.medium
 ---
 
+all_package_tests:
+  name: Run All Combinations of Editors/Platforms Tests
+  dependencies:
+  {% for editor in test_editors %}
```

Comment:
Not a huge deal, but we're inconsistent on the indentation of the `{% %}` blocks for nested loops. I.e. same indent for both here but increasing indent on line 50 (and same indent for the endfors on 79 and 80)